gulp-tasks/documentation.js

```diff
@@ -23,8 +23,8 @@ const {globPromise, taskHarness} = require('../utils/build');
 
 const FRONT_MATTER = `---
```

Comment:
You need to comment this value out to fix the lint errors. Test it's working with `gulp lint`.